### DISCUSSION TITLE
why block ? ,help !!!

### DISCUSSION THREAD

**Discussion by alizargar21** [vercel/community#1277] [Help] [answered]
### Summary

hello guys.i try deploy a template , but vercel blocked my account! .please help :(

**Comment by swarnava** [comment, marked as answer]
Please reach out to registrations@vercel.com using your registered email

**Reply by HMser** [reply]
My account also blocked, and I tried to contact registrations@vercel.com. no reply 💔

**Reply by swarnava** [reply]
They will soon. :)
